### PR TITLE
Update key value extension to get the json from remote URL

### PR DESCRIPTION
**Pull request by NirLevy98**
## Description

Improve the key-value extension to load the new key value from remote url
## Screencast

https://github.com/raycast/extensions/assets/7195780/7f82be0f-96eb-41c7-b73b-d78c2939449b

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by gkpln3** [comment]
Lgtm

File: extensions/key-value/package.json [hunk 1]
```diff
@@ -28,15 +28,46 @@
...
+          "title": "Key"
+        },
+        {
+          "name": "jsonKeyForValue",
+          "description": "Key name to get the value from",
+          "type": "textfield",
+          "required": true,
+          "title": "Value"
+        }
+      ]
```
Comment (pernielsentikaer) [inline extensions/key-value/package.json:59]: Should the new preferences be required?

Comment (NirLevy98) [reply extensions/key-value/package.json:59]: It's should be required only under the reload command

Comment (pernielsentikaer) [reply extensions/key-value/package.json:59]: Makes sense 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**